mlir/utils/jenkins/Jenkinsfile

```diff
@@ -304,18 +292,39 @@ pipeline {
                             steps {
                                 sh 'ldconfig'
                                 dir('MIOpen/build/') {
-                                    sh 'date --utc +%Y-%m-%d > tuning-date'
                                     sh """
                                     bash ${WORKSPACE}/mlir/utils/jenkins/miopen-tests/miopen_validate.sh --test-all --tuning\
                                        < ${WORKSPACE}/mlir/utils/jenkins/miopen-tests/resnet50-miopen-configs"""
+                                    sh 'date --utc +%Y-%m-%d > MIOpenUserDB/tuning-date'
                                     sh 'ls -l MIOpenUserDB'
                                 }
-                                // Save user database  for nightly jobs
+                                // Save user database for nightly jobs
                                 archiveArtifacts artifacts: 'MIOpen/build/MIOpenUserDB/**, MIOpen/build/tuning-date',\
                                                  onlyIfSuccessful: true
                             }
                         }
- 
+                        stage("Test selected MIOpen configs") {
```

Comment:
How long does this stage take and is there any possibility to make it part of PR CI if it is short enough?

Reply:
To rephrase, I'm also curious how long does it take for all configs to run through tuning/non-tuning? If it is 2 mins perf flavor, we have a total of 24 flavors than in total it should consume around 50 mins. It sounds do-able as part of nightly as well.

Reply:
> How long does this stage take and is there any possibility to make it part of PR CI if it is short enough?
20-30 minutes.

Reply:
> To rephrase, I'm also curious how long does it take for all configs to run through tuning/non-tuning? If it is 2 mins perf flavor, we have a total of 24 flavors than in total it should consume around 50 mins. It sounds do-able as part of nightly as well.

Tuning takes very long time to finish. 3 hours on public CI and 6 hours on private CI. non-tuning takes about 30 minutes to finish.

Reply:
20 - 30 mins does seem to be reasonable to add to PR CI, I'd prefer to do that so we have minimal coverage on PR submissions.

On the other hand, I wasn't talking about the tuning itself, but to test the tuned configs based on weekly result. That should be roughly the same amount of test on testing without perfdb.

Reply:
So just to be clear I'm proposing this:
 - PR test: test selected MIOpen configs
 - nightly test: test `MIOpen Resnet50 config Test`
 - Weekly test stay the same and do the actual tuning

Reply:
> 20 - 30 mins does seem to be reasonable to add to PR CI, I'd prefer to do that so we have minimal coverage on PR submissions.
> 
> On the other hand, I wasn't talking about the tuning itself, but to test the tuned configs based on weekly result. That should be roughly the same amount of test on testing without perfdb.

The performance tests are run with and without perfDb (only for NCHW). Each run (tuning or non-tuning) takes about 10 minutes, or the total of 20 minutes. 

Should we also do it for NHWC?

Reply:
> So just to be clear I'm proposing this:
> 
> * PR test: test selected MIOpen configs
> * nightly test: test `MIOpen Resnet50 config Test`
> * Weekly test stay the same and do the actual tuning

OK.

Reply:
Is it good enough if we do selected MIOpen config tests only on private CI?